[2024-04-16T10:37:17.673+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-16T10:37:17.760+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: priority_one_number_two_dag_v1.check_sum_number scheduled__2024-03-13T00:00:00+00:00 [queued]>
[2024-04-16T10:37:17.807+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: priority_one_number_two_dag_v1.check_sum_number scheduled__2024-03-13T00:00:00+00:00 [queued]>
[2024-04-16T10:37:17.817+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-16T10:37:17.885+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): check_sum_number> on 2024-03-13 00:00:00+00:00
[2024-04-16T10:37:17.921+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=588) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-16T10:37:17.925+0000] {standard_task_runner.py:63} INFO - Started process 593 to run task
[2024-04-16T10:37:17.930+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'priority_one_number_two_dag_v1', 'check_sum_number', 'scheduled__2024-03-13T00:00:00+00:00', '--job-id', '332', '--raw', '--subdir', 'DAGS_FOLDER/priority_one_number_two_dag.py', '--cfg-path', '/tmp/tmpicgg53fp']
[2024-04-16T10:37:17.938+0000] {standard_task_runner.py:91} INFO - Job 332: Subtask check_sum_number
[2024-04-16T10:37:18.160+0000] {task_command.py:426} INFO - Running <TaskInstance: priority_one_number_two_dag_v1.check_sum_number scheduled__2024-03-13T00:00:00+00:00 [running]> on host eaff2d74d54d
[2024-04-16T10:37:18.787+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ika' AIRFLOW_CTX_DAG_ID='priority_one_number_two_dag_v1' AIRFLOW_CTX_TASK_ID='check_sum_number' AIRFLOW_CTX_EXECUTION_DATE='2024-03-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-13T00:00:00+00:00'
[2024-04-16T10:37:18.796+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-16T10:37:19.127+0000] {logging_mixin.py:188} INFO - Total dari list number random : 591
[2024-04-16T10:37:19.129+0000] {logging_mixin.py:188} INFO - Odd Sum
[2024-04-16T10:37:19.134+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-16T10:37:19.135+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-16T10:37:19.163+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=priority_one_number_two_dag_v1, task_id=check_sum_number, execution_date=20240313T000000, start_date=20240416T103717, end_date=20240416T103719
[2024-04-16T10:37:19.267+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-16T10:37:19.366+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-16T10:37:19.377+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
